### ISSUE TITLE
Insufficient Contrast Ratio between Background and Foreground for Action Dropdown

### ISSUE THREAD

**Issue by maggiehua01** [CMU-313/fall-2021-hw2-crabs-adjust-humidity#1]
The accessibility score is currently 82 and gives the warning "Background and foreground colors do not have a sufficient contrast ratio." Low-contrast text in the Actions Toggle Dropdown ("Documents > Trash can" page) may be difficult or impossible for many users to read.
